build.sbt

```diff
@@ -89,6 +89,7 @@ lazy val sdkScala = project
       "scalaVersion" -> scalaVersion.value),
     buildInfoPackage := "com.akkaserverless.scalasdk",
     Compile / scalacOptions ++= Seq("-release", "8"),
+    Compile / javacOptions ++= Seq("--release", "8"),
```

Comment:
we should test at least a few of the samples in CI with JDK 8

Reply:
or just drop support for 8: https://github.com/lightbend/akkaserverless/discussions/5194